src/strategies/cake-pools/index.ts

```diff
@@ -0,0 +1,72 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'pancake-swap';
+export const version = '0.0.1';
+
+const sousChefabi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'userInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses: string[],
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const sousBalances = await Promise.all(
+    options.chefAddresses.map((item) =>
+      multicall(
+        network,
+        provider,
+        sousChefabi,
+        addresses.map((address: any) => [
+          item.address,
+          'userInfo',
+          [address],
+          { blockTag }
+        ]),
+        { blockTag }
+      )
+    )
+  );
```

Comment:
Hi @chef-jojo Can you add a limit to chefAddresses to allow only 5 requests per one strategy, a space can use multiple strategies if required :)
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy